harvard-lts/drs-import-management-service
# JWT Public key loaded from file and serialized

**JWT Public key loaded from file and serialized**
* * *

**GitHub Issue**: ([link](https://app.zenhub.com/workspaces/hdc-62163f60fb08c60011fa4795/issues/harvard-lts/hdc/195))

# What does this Pull Request do?
Changes the JWT public key loading to be from a file. Also, serialization is applied to the key with the cryptography library.

# How should this be tested?

A description of what steps someone could take to:
* Make sure test .env variables are updated based on .env.example files
* Rebuild test environment: docker-compose -f docker-compose-test.yml build --no-cache
* Run the local test environment: `docker-compose -f docker-compose-test.yml up`
* Execute functional tests: `docker exec -it test_runner pytest test/functional`

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- functional tests Yes
- unit tests No
- integration tests No

# Interested parties
@jessjass

Keyword: resolved
